pep-0617.rst

```diff
@@ -571,23 +581,23 @@ different possibilities:
 If the action is omitted and Python code is being generated, then a list
 with all the parsed expressions get returned (this is meant for debugging).
 
-As an illustrative example this simple grammar file allows to directly generate a full
-parser that can parse simple aritmetic expressions and that returns a valid Python AST:
+As an illustrative example this simple grammar file allows to directly
+generate a full parser that can parse simple arithmetic expressions and that
+returns a valid C-based Python AST:
 
 ::
 
-    start[mod_ty]: a=stmt* $ { Module(a, NULL, p->arena) }
-    stmt[stmt_ty]: a=expr_stmt { a }
-    expr_stmt[stmt_ty]: a=expression NEWLINE { _Py_Expr(a, EXTRA) }
-    expression[expr_ty]: ( l=expression '+' r=term { _Py_BinOp(l, Add, r, EXTRA) }
-                        | l=expression '-' r=term { _Py_BinOp(l, Sub, r, EXTRA) }
+    start[mod_ty]: a=expr_stmt* $ { Module(a, NULL, p->arena) }
+    expr_stmt[stmt_ty]: a=expr NEWLINE { _Py_Expr(a, EXTRA) }
+    expr[expr_ty]: ( l=expr '+' r=term { _Py_BinOp(l, Add, r, EXTRA) }
+                        | l=expr '-' r=term { _Py_BinOp(l, Sub, r, EXTRA) }
                         | t=term { t }
                         )
```

Comment:
Maybe convert all rules that don't fit on a line to the new format, with each alternative starting with `|` on a new line (after 4 spaces)?

Reply:
Done in 00c739a